lib/commands/basic/init.js

```diff
@@ -66,30 +66,6 @@ let fetchTemplate = (template) => {
   }, 5000);
 
   fs.writeFileSync(`./${projectName}/mevn.json`, projectConfig.join('\n').toString());
-
-  if (template === 'nuxt') {
-    setTimeout(() =>{
-
-      console.log('\n');
-
-      inquirer.prompt([{
-        name: 'mode',
-        type: 'list',
-        message: 'Choose your preferred mode',
-        choices: ['Universal', 'SPA']
-      }])
-      .then((choice) => {
-        if (choice.mode === 'Universal') {
```

Comment:
It is good to have the `pwa required` prompt here as from the previous commit :+1: